clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactory.java

```diff
@@ -191,25 +220,50 @@ public void destroy(final String identifier) throws EhcacheEntityNotFoundExcepti
     if (existingMaintenance == null) {
       localMaintenance = createAccessLockFor(identifier).tryWriteLock();
     }
+
     if (existingMaintenance == null && localMaintenance == null) {
       throw new EntityBusyException("Destroy operation failed; " + identifier + " clustered tier's maintenance lease held");
     } else {
-      try {
         EntityRef<EhcacheClientEntity, UUID> ref = getEntityRef(identifier);
         try {
           if (!ref.destroy()) {
             throw new EntityBusyException("Destroy operation failed; " + identifier + " clustered tier in use by other clients");
           }
         } catch (EntityNotProvidedException e) {
-          LOGGER.error("Unable to delete clustered tier manager for id {}", identifier, e);
+          LOGGER.error("Unable to delete clustered tier manager for id " + identifier, e);
```

Comment:
I forgot to remove it I think. One note on the standard pattern though. The curly braces are useful to prevent string concatenation upfront. So we should use them when we know that most of the time, the log will be ignored. This is the case for DEBUG for instance. However, for an ERROR, we could argue that it's not a good idea. Because looking for curly braces in a string have a huge performance impact. So String concatenation is more appropriate. That said, errors are not frequent. So it is really more dodgy for INFO.

The same applies when you call complicated methods to generate a log. This is be called upfront. So you better shield the call with a `isInfoEnabled` and then use String concatenation.